docs/src/main/asciidoc/hibernate-orm-guide.adoc

```diff
@@ -179,6 +179,9 @@ How many rows are fetched at a time by the JDBC driver.
 `quarkus.hibernate-orm.jdbc.statement-batch-size`::
 The number of updates (inserts, updates and deletes) that are sent by the JDBC driver at one time for execution.
 
+`quarkus.hibernate-orm.globally-quoted-identifiers`:: (defaults to `false`)
+Whether Hibernate should escape all identifiers.
```

Comment:
should it say "quote" rather than "escape" ? It's not the same.

Reply:
Fixed. Thanks for remarks.

Reply:
Resolved in  https://github.com/quarkusio/quarkus/pull/3057/commits/c0efb3e277c69bccce1b4e3e9abac2eb9b426bec

Reply:
> should it say "quote" rather than "escape" ? It's not the same.

Resolved in https://github.com/quarkusio/quarkus/commit/c0efb3e277c69bccce1b4e3e9abac2eb9b426bec